.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?
+- [ ] Have you verified that you are running the latest version of Nancy?
+- [ ] Have you verified if the problem exist in both `debug` and `release` mode?
+- [ ] Have you searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported?
+
+### Description
+[A description of the bug or feature]

```

Comment:
> you are talking about HTML5 placeholders, right?

Nah, I'm just talking about HTML comments.

> I kind of like it being actual text as well, it will force people to actively replace it :D

The problem is that a lot of people won't, so we'll get back loads and loads of issues saying "Thanks for reporting the issue to Nancy!" at the bottom :stuck_out_tongue_closed_eyes: